app/controllers/settings_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SettingsController < ApplicationController
+  before_action :require_current_user
+
+  def edit_profile
+  end
+
+  def update_profile
```

Comment:
Assignment Branch Condition size for update_profile is too high. [20.2/20]<br>Method has too many lines. [11/10]